src/assets/data/markdown/ionic-weather-v4-step08.md

```diff
@@ -4,19 +4,40 @@ Cordova plugins are used when you want to access native functionality from your
 
 In this lab you will learn how to:
 
-- Install Cordova Plugins
-- Use Ionic Native wrappers
+- Install a Cordova plugin
+- Use an Ionic Native wrapper for a particular Cordova plugin
 - Create a non-HTTP service
-- Use promises
-- Use promises and observables together by mapping the promises to observables
-- Use `flatMap` to essentially "chain" observables
+- Use Promises and Observables
+- Map a Promise to an Observable
+- Use `flatMap` to "chain" observables
 - Keep complex code clean
 
-## Geo-Location
+## Using Geolocation
 
-Right now, the application gives us the weather for a specific location. It would be nice if it could let us know what the weather is for our current location. Checking the <a href="https://ionicframework.com/docs/native/" target="_blank">Ionic Native</a> project page we see that a plugin and Ionic Native wrapper exists for geolocation.
+Right now, the application gives us the weather for a specific location. It would be nice if it could let us know what the weather is for our current location. Checking the <a href="https://ionicframework.com/docs/native/" target="_blank">Ionic Native</a> project page, we see that a plugin and Ionic Native wrapper exists for geolocation.
 
-### Install the Plugin
+## Comparing the Cordova Plugin and Ionic Native Wrapper
```

Comment:
I don't want to put a ton of information like this in the labs. I have been completely re-writing them, AAMOF, to make them more step by step things they do without all this extraneous information. This sort of stuff is covered by the lecture.